finetuner/data.py

```diff
@@ -232,12 +257,15 @@ class CSVContext:
 
     def __init__(
         self,
-        model: str,
+        model: Optional[str] = None,
+        task: Optional[str] = None,
```

Comment:
Is this task parameter used anywhere? I can not any code which using it. Beside this, it is also not documented in the docstring.